options_test.go

```diff
@@ -0,0 +1,40 @@
+package envbuilder_test
+
+import (
+	"testing"
+
+	"github.com/coder/envbuilder"
+	"github.com/coder/serpent"
+	"github.com/stretchr/testify/require"
+)
+
+type testEnvCase struct {
+	Env      string
+	EnvValue string
+	Expected interface{}
+	Actual   func() interface{}
+}
+
+func TestEnvOptions(t *testing.T) {
+	t.Setenv("SETUP_SCRIPT", "setup.sh")
+	t.Setenv("CACHE_TTL_DAYS", "7")
+	t.Setenv("IGNORE_PATHS", "/var,/tmp")
+	t.Setenv("SKIP_REBUILD", "true")
+	t.Setenv("GIT_CLONE_SINGLE_BRANCH", "")
+
+	var o = envbuilder.Options{}
```

Comment:
Nit
```suggestion
	var o envbuilder.Options
```

It's currently somewhere between `o := envbuilder.Options{}` and the suggestion above
We should pick one, I don't mind which